### PR TITLE
App ID, name and splashscreen up to env

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,18680 @@
+{
```
Comment (KedziaPawel) [inline package-lock.json:1]: we use yarn

File: android/settings.gradle [hunk 2]
```diff
@@ -1,4 +1,6 @@
-rootProject.name = 'GoldWallet'
+rootProject.name = 'Testnet Gold Wallet'
```
Comment (KedziaPawel) [inline android/settings.gradle:1]: shouldn't be this in env?

File: android/.project [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>GoldWallet</name>
+	<name>GoldWalletss</name>
```
Comment (KedziaPawel) [inline android/.project:3]: typo i guess

File: App.tsx [hunk 4]
```diff
@@ -33,7 +33,7 @@ export default class App extends React.PureComponent {
     return (
       <I18nextProvider i18n={i18n}>
         <Provider store={store}>
-          <AppStateManager handleAppComesToBackground={this.lockScreen} />
+          <AppStateManager handleAppComesToForeground={this.lockScreen} />
```
Comment (KedziaPawel) [inline App.tsx:36]: what is the reason for this change? with this, you can hack the pin screen by clicking real fast a tab bar when app is foregrounding

File: android/app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -11,7 +11,7 @@
         <item name="android:background">@drawable/background_splash</item>
     </style>
     <style name="SplashScreen">
-    <item name="android:src">@mipmap/splash_beta</item>
+    <item name="android:src">@mipmap/splash</item>
```
Comment (KedziaPawel) [inline android/app/src/main/res/values/styles.xml:14]: will the splash screen be changed in the bitrise during deployment proccess?

**Review by KedziaPawel — COMMENTED** [review]
why are we moving envs files to the config folder? the .env file is in the root of the project so I think it is ok to keep them next to each other

**Comment by Kamill90** [comment]
> why are we moving envs files to the config folder? the .env file is in the root of the project so I think it is ok to keep them next to each other

it is easier to have version-specific env and just run 'android:prod' than modifying env and running 'android:prod'

File: android/settings.gradle (hunk 2 above)
Comment (Kamill90) [reply android/settings.gradle:1]: No, but should be 'GoldWallet' indeed.

File: android/.project (hunk 3 above)
Comment (Kamill90) [reply android/.project:3]: verifying if it's needed. It's not

File: App.tsx (hunk 4 above)
Comment (Kamill90) [reply App.tsx:36]: It was a quick workaround for the beta, but ok, I will sit to it once more

File: android/app/src/main/res/values/styles.xml (hunk 5 above)
Comment (Kamill90) [reply android/app/src/main/res/values/styles.xml:14]: yes

**Comment by KedziaPawel** [comment]
> > why are we moving envs files to the config folder? the .env file is in the root of the project so I think it is ok to keep them next to each other
> 
> it is easier to have version-specific env and just run 'android:prod' than modifying env and running 'android:prod'

I'm talking about these changes, also I'm not sure if the .env.test will work properly if it won't be in the root
Zrzut ekranu 2020-10-7 o 13 00 09

File: app.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {
-  "name": "GoldWallet",
-  "displayName": "GoldWallet"
-}
\ No newline at end of file
+  "name": "Testnet Gold Wallet",
```
Comment (KedziaPawel) [inline app.json:2]: shouldn't be still GoldWallet?

**Comment by KedziaPawel** [comment]
> > > why are we moving envs files to the config folder? the .env file is in the root of the project so I think it is ok to keep them next to each other
> > 
> > 
> > it is easier to have version-specific env and just run 'android:prod' than modifying env and running 'android:prod'
> 
> I'm talking about these changes, also I'm not sure if the .env.test will work properly if it won't be in the root
> Zrzut ekranu 2020-10-7 o 13 00 09

**Review by KedziaPawel — APPROVED** [review]
